android/app/google-services.json

```diff
@@ -14,6 +14,22 @@
         }
       },
       "oauth_client": [
+        {
```

Comment:
I don't think we will need to attach the json file in the future since our SHA-1  keys are different
But still it shouldn't be the case where u needed to insert your json file since the Gsignin worked previously too